### PR TITLE
Allow popular and stations to add to queue and playlist

### PR DESCRIPTION
**Pull request by darenang44**

### CODE REVIEW

File: client/app/pod/popularController.js [hunk 1]
```diff
@@ -28,16 +31,16 @@ angular.module('throwcast.popular')
     });
   };
 
-  $scope.unsubscribe = function (userId, stationId) {
-    $http.delete('api/user/' + userId + '/subscriptions/' + subId).then(function (res) {
-      $scope.message = 'Unsubscribed to ' + res.data.name + '.';
+  $scope.addToQueue = function (userId, podcastId) {
+    $http.post('api/user/' + userId + '/queue/' + podcastId).then(function (res) {
```
Comment (PrinceOm) [inline client/app/pod/popularController.js]: Add podcastId to req.body instead.

File: client/app/pod/stationsController.js [hunk 2]
```diff
@@ -34,4 +35,33 @@ angular.module('throwcast.stations')
       $scope.stationPodcasts = stationPodcasts;
     });
   };
+
+  $scope.addToQueue = function (userId, podcastId) {
+    $http.post('api/user/' + userId + '/queue/' + podcastId).then(function (res) {
```
Comment (PrinceOm) [inline client/app/pod/stationsController.js]: Add podcastId to req.body.

File: client/app/pod/stationsController.js [hunk 3]
```diff
@@ -34,4 +35,33 @@ angular.module('throwcast.stations')
...
+
+  $scope.unsubscribe = function (userId, stationId) {
+    $http.delete('api/user/' + userId + '/subscriptions/' + subId).then(function (res) {
+      $scope.message = 'Unsubscribed to ' + res.data.name + '.';
+      $scope.getStations();
+    });
+  };
+
+  $scope.subscribe = function (userId, stationId) {
+    $http.post('api/user' + userId + '/subscriptions' + stationId).then(function (res) {
```
Comment (PrinceOm) [inline client/app/pod/stationsController.js]: Add stationId to req.body

File: client/app/pod/popularController.js (hunk 1 above)
Comment (darenang44) [inline client/app/pod/popularController.js]: Done

File: client/app/pod/stationsController.js (hunk 2 above)
Comment (darenang44) [inline client/app/pod/stationsController.js]: Done

File: client/app/pod/stationsController.js (hunk 3 above)
Comment (darenang44) [inline client/app/pod/stationsController.js]: Done

**Pull request merged**